app/models/release_artifact.rb

```diff
@@ -267,7 +289,7 @@ class ReleaseArtifact < ApplicationRecord
                .uploaded,
          )
          .reorder(
-           created_at: DEFAULT_SORT_ORDER,
+           "#{table_name}.created_at": DEFAULT_SORT_ORDER,
```

Comment:
why?

Reply:
Reason: unions don't always know what table they're for, since a union could be across tables.